stream-transformation-test/stream-transformation-it/src/main/java/uk/gov/justice/framework/tools/transformation/DatabaseUtils.java

```diff
@@ -19,11 +20,11 @@
 
     private TestEventLogJdbcRepository eventLogJdbcRepository;
     private TestEventStreamJdbcRepository eventStreamJdbcRepository;
-
+    private EventFetcherRepository eventFetcherRepository = new EventFetcherRepository();
     private final LiquibaseUtil liquibaseUtil = new LiquibaseUtil();
-
+    DataSource dataSource = null;
```

Comment:
Should be private (and final if possible)

Reply:
private not final used outside constructor